src/solvers/smt2/smt2_dec.cpp

```diff
@@ -148,32 +148,32 @@ decision_proceduret::resultt smt2_dect::read_result(std::istream &in)
 
     const auto &parsed = parsed_opt.value();
 
-    if(parsed.id()=="sat")
-      res=resultt::D_SATISFIABLE;
-    else if(parsed.id()=="unsat")
-      res=resultt::D_UNSATISFIABLE;
+    if(parsed.id() == "sat")
+      res = resultt::D_SATISFIABLE;
+    else if(parsed.id() == "unsat" || parsed.id() == "unknown")
```

Comment:
:pick: Seems like a fairly dubious change best left in the `smt-fast-response` branch only, rather than added to develop.